src/ol/control/attribution.js

```diff
@@ -182,11 +182,21 @@ ol.control.Attribution.prototype.getSourceAttributions = function(frameState) {
       } else {
         intersectsTileRange = false;
       }
+      var attributionAlreadyAdded = false;
       if (intersectsTileRange) {
         if (sourceAttributionKey in hiddenAttributions) {
           delete hiddenAttributions[sourceAttributionKey];
         }
-        attributions[sourceAttributionKey] = sourceAttribution;
+        for (var existingKey in attributions) {
+          if (attributions.hasOwnProperty(existingKey)) {
```

Comment:
We create the `attributions` object and don't guard against people adding enumerable properties to `Object.prototype` elsewhere, so this `hasOwnProperty()` call is not necessary.